lib/checks/keyboard/inserted-into-focus-order.js

```diff
@@ -0,0 +1,2 @@
+return (axe.commons.dom.isFocusable(node) && node.tabIndex > -1 &&
```

Comment:
Why does this file exist? It's not a check you're using anywhere. I'm guessing you started here and then moved it into a commons?

Reply:
Yeah I was honestly moving all kinds of things around trying to figure out why tests wouldn't run, turns out it was because I had left the "all" and "none" fields out of the rule definition -- thanks so much for catching that -- i'll clean this up and resolve your comments and get it back to you